app/src/main/java/app/insti/Utils.java

```diff
@@ -193,6 +193,10 @@ public static void openUserFragment(User user, Fragment currentFragment, View sh
         );
     }
 
+    public static void openUserFragment(String userId, FragmentActivity fragmentActivity) {
+        updateFragment(UserFragment.newInstance(userId), fragmentActivity);
```

Comment:
This method is almost duplicate, but I guess that's okay

Reply:
Actually, you could just call this from the other overload which takes `User` and `FragmentActivity` and make this the primary one

Reply:
I planned on changing the other method to use UserFragment$newInstance(User minUser). I'll change this for now and make a separate PR for that.